tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests.csproj

```
@@ -24,8 +24,8 @@
     <EmbeddedResource Include="TestFiles\TestFile.txt" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="../../src/System.IO.Abstractions/System.IO.Abstractions.csproj" />
-    <ProjectReference Include="../../src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj" />
+    <ProjectReference Include="../../src/TestableIO.System.IO.Abstractions/STestableIO.ystem.IO.Abstractions.csproj" />
```

```suggestion
    <ProjectReference Include="../../src/TestableIO.System.IO.Abstractions/TestableIO.System.IO.Abstractions.csproj" />
```

Typo in the project reference!

---

Ah, great catch. Weird that CI did not complain about this 🤔